imports/client/components/RelativeTime.tsx

```diff
@@ -21,7 +21,18 @@ const RelativeTime = ({
     return () => {
       Meteor.clearTimeout(timeout);
     };
-  }, [date, formatted.millisUntilChange, maxElements, minimumUnit, now, terse]);
+  }, [
+    date,
+    formatted.millisUntilChange,
+    maxElements,
+    minimumUnit,
+    now,
+    terse,
+    // Note that we explicitly include formatted.formatted here so that we set a
+    // new timeout if the formatted string changes but (by change) the
+    // millisUntilChange does not.
+    formatted.formatted,
```

Comment:
Ideally this would have been fixup'd into the previous commit in the series, but whatever